components/create/WorkHistory.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import CreatePortfolioContext from 'components/create/CreatePorfolioContext';
+import { Stack } from '@mui/material';
+import { useContext, useRef, RefObject } from 'react';
+import { StyledTextField, StyledButton } from 'components/create/Styled';
+const WorkHistory = () => {
+  const formRef = useRef() as RefObject<HTMLFormElement>;
+  const entry = {
+    position: '',
+    company: '',
+    dateWorked: '',
+    description: '',
+  };
+  const { work, setWork } = useContext(CreatePortfolioContext);
+  const onSubmit = async () => {
+    work.push(entry);
+    setWork([...work]);
+    formRef.current?.reset();
+  };
+  return (
+    <form ref={formRef}>
+      <Stack direction="column" spacing={1}>
+        <StyledTextField
+          label={'Position'}
+          onChange={(element) => {
+            entry.position = element.target.value;
+          }}
+        />
+        <StyledTextField
+          label={'Company'}
+          onChange={(element) => {
+            entry.company = element.target.value;
+          }}
+        />
+        <StyledTextField
+          label={'Interval of employement'}
+          onChange={(element) => {
+            entry.dateWorked = element.target.value;
+          }}
+        />
+        <StyledTextField
+          label={'Summary of what you did'}
```

Comment:
This can just be `label="Summary of what you did`"

Reply:
fixed

Reply:
fixed